### PR TITLE
Lwms tile layer

### LINKED ISSUE #20: [Component] LwmsTileLayer
To migrate a component from Vue2Leaflet to vue-leaflet (and so from vue2 to vue3) follow this steps:

- create a function in `functions` folder called `componentName.js` (without the `L`)
- check what mixins are used in the vue2 version and see if all the relative composition API function are present, if not they need to be created
  - Each composition API function needs to export two objects: props and setup, see `marker.js` for an example.
- create a component in `components` folder called `LComponentName.vue` 
- follow the guide of `LMarker.vue` to understand how to deal with reactivity, dynamic import and SSR
- add and test the component inside `playground.vue`

### PR DESCRIPTION
**Pull request by DonNicoJs**
Fixes #20

### CODE REVIEW

**Comment by DonNicoJs** [comment]
@mikeu Would you mind taking a look at this? After this is merged I think I am going to do a minor release with all the new components!

File: src/Playground.vue [hunk 1]
```diff
@@ -1,74 +1,105 @@
...
+        <l-wms-tile-layer
+          v-for="layer in wmsLayers"
+          :key="layer.name"
+          :base-url="baseUrl"
+          :layers="layer.layers"
+          :visible="layer.visible"
+          :name="layer.name"
+          layer-type="base"
+          >></l-wms-tile-layer
+        >
```
Comment (mikeu) [inline src/Playground.vue:100]:
```suggestion
        ></l-wms-tile-layer>
```
Minor formatting and not having an extra `>` as the layer contents

File: src/Playground.vue [hunk 2]
```diff
@@ -1,74 +1,105 @@
...
-        </l-popup>
-      </l-rectangle>
-    </l-map>
-    <button @click="changeIcon">New kitten icon</button>
+        <l-wms-tile-layer
+          v-for="layer in wmsLayers"
+          :key="layer.name"
+          :base-url="baseUrl"
+          :layers="layer.layers"
+          :visible="layer.visible"
```
Comment (mikeu) [inline src/Playground.vue:96]:
```suggestion
          :visible="layer.visible"
          :format="layer.format"
          :transparent="layer.transparent"
```
Demo more available options

**Review by mikeu — APPROVED** [review]
LGTM!

I think at some point I might submit a PR to switch the playground back to only a single map (unless you prefer having both of them?) but this is great for now, and of course the component is the actually important part :)

**Comment by DonNicoJs** [comment]
@mikeu I think we can switch to a single map, or even better a barebone playground as soon as we have the docs running! Merging this! Thank you!

**Pull request merged**